src/dialect/mod.rs

```diff
@@ -782,6 +782,11 @@ pub trait Dialect: Debug + Any {
     fn supports_insert_set(&self) -> bool {
         false
     }
+
+    /// Does the dialect support table function in insertion?
+    fn supports_table_function_in_insertion(&self) -> bool {
```

Comment:
```suggestion
    fn supports_insert_table_function(&self) -> bool {
```
maybe something like this to match the style of `supports_insert_set` above?